templates/terraform/.github/workflows/auto-context.yml

```diff
@@ -0,0 +1,54 @@
+name: "auto-context"
+on:
+  schedule:
+  # Update context.tf nightly
+  - cron:  '0 3 * * *'
+
+jobs:
+  update:
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Update context.tf
+      shell: bash
+      id: update
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      run: |
+        if [[ -f context.tf ]]; then
+          echo "Discovered existing context.tf! Fetching most recent version to see if there is an update."
+          curl -o context.tf -fsSL https://raw.githubusercontent.com/cloudposse/terraform-null-label/master/exports/context.tf
+          if git diff --no-patch --exit-code context.tf; then
+            echo "No changes detected! Exiting the job..."
+          else
+            echo "context.tf file was updated. Need to rebuild README.md."
+            make init
+            make readme/build
+            echo "::set-output name=create_pull_request=true"
```

Comment:
Technically unnecessary because `create-pull-request` will only open a PR if there were changes.

Reply:
`create-pull-request` will still do a lot of work to determine that no changes were needed, and since we will be running over 100 of these in parallel, and the normal case will be no changes, I want to short-circuit as much as possible.